### PR TITLE
Address CoSWID to CoMID relations

### LINKED ISSUE #41: Discuss and define relationship between CoMID and CoSWID
While the current draft documents adequately the relationship between COMID and CoMID, however  the relationship which describes how CoMID is linked to CoSWID and also how CoSWID links to CoMID is not documented.

This not only needs an update to COMID CDDL, but also on the CoSWID CDDL, which will have an updated relation types to CoMID.

### PR DESCRIPTION
**Pull request by yogeshbdeshpande**
Fixes #41

### CODE REVIEW

File: draft-birkholz-rats-corim.md [hunk 1]
```diff
@@ -178,6 +178,38 @@ Not included in the full CDDL definition are CDDL dependencies to CoSWID. The fo
...
+
+The link-entry map has been extended to define new relationship between a
+CoSWID and CoMID
+
+When defining CoSWID to CoMID link entry map the following optional elements
+should not be used:
+* artifact
+* ownership
+* use
+
```
Comment (nedmsmith) [inline draft-birkholz-rats-corim.md:193]: The ISO SWID TAGs spec also allows use of IANA link relations IANA (http:// www .iana .org/ assignments/ link -relations/ link -relations .xhtml) which we want to disallow for use with CoSWID to CoMID.

**Review by nedmsmith — APPROVED** [review]
I approve of the proposed changes with consideration for the item I commented on.

File: draft-birkholz-rats-corim.md (hunk 1 above)
Comment (yogeshbdeshpande) [reply draft-birkholz-rats-corim.md:193]: Ok, I will add this specific clarification in the text!

File: draft-birkholz-rats-corim.md [hunk 2]
```diff
@@ -178,6 +178,42 @@ Not included in the full CDDL definition are CDDL dependencies to CoSWID. The fo
...
+The new rel values (as listed section 4.4) will not be defined in ISO/IEC 19770-2:2015
+[SWID] specification. Hence for modelling CoSWID to CoMID relations
+ISO/IEC 19770-2:2015 [SWID] specification shall not be used.
+
+# CoSWID Indexed Label Values
+
+# 4.4. Link Rel Values
```
Comment (thomas-fossati) [inline draft-birkholz-rats-corim.md:200]: spurious "4.4.": section numbering is automated

File: draft-birkholz-rats-corim.md [hunk 3]
```diff
@@ -178,6 +178,42 @@ Not included in the full CDDL definition are CDDL dependencies to CoSWID. The fo
...
+ | Index | Relationship Type Name | Definition                                     |
+ --------|----------------------- |--------------------------------------------|
+ | 12    | m-requires             | The link references a prerequisite module  |
+ |       |                        | tag(s) to be loaded or present             |
+ |       |                        | for installing this software.       |      |
+ | 13    | m-runs-on              | The link references a module tag that      |
+ |       |                        | this software run on                       |
+
+New indexes 12 and 13 will be registered in the
+IANA "Software Tag Link Relationship Values" registry defined in Section 6.2.7.
```
Comment (thomas-fossati) [inline draft-birkholz-rats-corim.md:215]: hardcoded references to section (i.e., "Section 6.2.7.") is going to break.  You need to add an anchor in the referenced section and put a link here to that.

**Pull request merged**